packages/react-searchbox/src/components/SearchBox.js

```diff
@@ -337,13 +337,16 @@ class SearchBox extends React.Component {
   };
 
   onSuggestionSelected = suggestion => {
+    if (!suggestion) {
+      return;
```

Comment:
@mohdashraf010897 We can't ignore it. Clear the value and trigger the custom and default queries.

Reply:
> if (!suggestion) {
      this.componentInstance.setValue('', {
        triggerDefaultQuery: true,
        triggerCustomQuery: true,
        stateChanges: true
      });
      return;
    }